docs/release-notes/highlights-1.2.0.asciidoc

```diff
@@ -0,0 +1,43 @@
+[[release-highlights-1.2.0]]
+== 1.2.0 release highlights
+
+[float]
+[id="{p}-120-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 1.2.0 of {n}. See <<release-notes-1.2.0>> for the full list of changes.
+
+[float]
+[id="{p}-120-enterprise-search"]
+==== Support for Elastic Enterprise Search
+
+ECK 1.2.0 introduces a new Custom Resource Definition (CRD) to easily deploy and orchestrate link:https://www.elastic.co/enterprise-search[Elastic Enterprise Search] on Kubernetes. With just a few lines of YAML configuration, you can launch an instance of link:https://www.elastic.co/app-search/[App Search] or link:https://www.elastic.co/workplace-search[Workplace Search] (requires an Enterprise license or a <<{p}-licensing,trial>>) and connect it to an Elasticsearch cluster.
+
+
+[float]
+[id="{p}-120-beats"]
+==== Support for Beats
+
+Deploying link:https://www.elastic.co/beats/[Beats] on Kubernetes is now much easier using the new Beats CRD. You can deploy and manage data shippers such as Filebeat, Metricbeat, Auditbeat and others using ECK and effortlessly connect them to Elasticsearch and Kibana for storage and analysis.
+
+
+[float]
+[id="{p}-120-apm-agent-configuration"]
+==== Support for APM Agent central configuration
+
+link:https://www.elastic.co/guide/en/kibana/current/agent-configuration.html#agent-configuration[APM Agent central configuration] with ECK-managed Kibana and APM Servers is easier to configure. Simply add `kibanaRef` to the APM Server configuration and ECK will inject the required configuration to connect APM Server to the referenced Kibana.
+
+
+[float]
+[id="{p}-120-distroless"]
+==== Smaller and more secure operator image
+
+The operator container image is now much smaller and more secure thanks to switching to link:https://github.com/GoogleContainerTools/distroless[Distroless] as the base image.
+
+
+[float]
+[id="{p}-120-known-issues"]
+=== Known issues
+
+- Elasticsearch link:https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html#transform-node[transform nodes] are not automatically labelled by the operator. If you need to identify transform nodes, use the `podTemplate` of the `nodeSet` to add your own label.
+- If you want to try out Workplace Search with a trial, use the <<{p}-licensing,ECK trial licensing>> mechanism instead of using the trial button provided on screen.
```

Comment:
Do we need to specify which screen we refer to?

Reply:
What would be a good way to phrase it? 
_Start Trial button on the Enterprise Search splash screen_?

Reply:
That sounds much clearer (at least to me)